lib/src/i18n_model.dart

```diff
@@ -979,6 +980,42 @@ final _i18nModel = <LocaleType, Map<String, Object>>{
     'am': 'AM',
     'pm': 'PM'
   },
+  LocaleType.isl: {
+    'cancel': 'Hætta við',
+    'done': 'Klára',
+    'today': 'Í dag',
+    'monthShort': [
+      'Jan',
+      'Feb',
+      'Mar',
+      'Apr',
+      'Maí',
+      'Jún',
+      'Júl',
+      'Ágú',
+      'Sep',
+      'Okt',
+      'Nóv',
+      'Des'
+    ],
+    'monthLong': [
+      'Janúar',
+      'Febrúar',
+      'Mars',
+      'Apríl',
+      'Maí',
+      'Júní',
+      'Júlí',
+      'Ágúst',
+      'September',
+      'Október',
+      'Nóvember',
+      'Desember'
+    ],
+    'day': ['Mán', 'Þri', 'Mið', 'Fim', 'Fös', 'Lau', 'Sun'],
```

Comment:
Month & day names are not capitalized in Icelandic.
AFAICT, this should be reflected in i18n.

Reference:
https://meta.wikimedia.org/wiki/Capitalization_of_Wiktionary_pages#Capitalization_of_month_names

```suggestion
    'monthShort': [
      'jan',
      'feb',
      'mar',
      'apr',
      'maí',
      'jún',
      'júl',
      'ágú',
      'sep',
      'okt',
      'nóv',
      'des'
    ],
    'monthLong': [
      'janúar',
      'febrúar',
      'mars',
      'apríl',
      'maí',
      'júní',
      'júlí',
      'ágúst',
      'september',
      'október',
      'nóvember',
      'desember'
    ],
    'day': ['mán', 'þri', 'mið', 'fim', 'fös', 'lau', 'sun'],
```

----


I don't have a strong opinion on `am`/`pm` but I've never seen it abbreviated.
It could either be unabbreviated (`fyrir hádegi`/`eftir hádegi`) or omitted as has been done in other languages here (``/``).
But it can be kept as is for now.